README.md

```diff
@@ -155,19 +155,39 @@ modules in `components`, and `warn` level for the remaining codebase:
 RUST_LOG=casper_node::components::small=trace,casper_node::comp=info,warn
 ```
 
+### Logging network messages and tracing events
+
+Special logging targets exist in `net_in` and `net_out` which can be used to log every single network message leaving or
+entering a node when set to trace level:
+
+```
+RUST_LOG=net_in::TRACE,net_out::TRACE
```

Comment:
Should we make it 
```bash
RUST_LOG=$RUST_LOG:net_in::TRACE,net_out::TRACE
```
?
Otherwise someone might accidentally overwrite previous log levels when trying to add this special logging.

Reply:
I would keep it simple as is, this is just an example. I hope the user is competent enough to realize how to use `RUST_LOG` to compose things (otherwise, why are they setting these settings at all?)

Also, depending on how and where they set these settings, the `$RUST_LOG` might not even be the appropriate way to include previous vars.